app.js

```diff
@@ -280,16 +280,16 @@ app.use(function (req, res, next) {
     csrf: req.csrfToken(),
     navigation: navigation,
     gettext: req.gettext,
-    campaignHeader: app.locals.flags.campaign ? "" + (Math.floor(Math.random() * +app.locals.flags.campaign)) : 0
+    campaignHeader: app.locals.flags.campaign ? '' + (Math.floor(Math.random() * +app.locals.flags.campaign)) : 0
```

Comment:
Not sure how to test? "campaignHeader" doesn't seem to appear in the repo and I'm not sure what it's doing yet.